### ISSUE TITLE
tabwrite~ Not working?

### ISSUE THREAD

**Issue by g2oo7fy** [monkeyswarm/MobMuPlat#80]
Hey,

Before complaining I want to thank you for your great work. Its really fun to play with MobMuPlat and I'm getting closer to using it in serious work.

I tried several setups with tabwrite~ but it seems to me that there is no connection between the object and the table. Patches that work nice and smoothly on a desktop computer don't work at all in the native GUI on an iPad. Setting values in the table directly works graphically and using messages. Only tabwrite~ doesn't.

Any idea?

Best Björn

**Comment by g2oo7fy** [comment]
yes, I can draw into a table. It is only the tabwrite~ object that does not behave as expected. I checked you example patch too, but there you use soundfiler to update the table. I also found other examples in your patches where you use tabwrite, but not the tilde object.

**Comment by monkeyswarm** [comment]
Can you post a patch which shows the discrepancy in behavior?

**Comment by g2oo7fy** [comment]
Sure

[simpleSpec.zip](https://github.com/monkeyswarm/MobMuPlat/files/9165555/simpleSpec.zip)

In this patch there is another issue on my iPad: There is no visible change in the state of the toggle. That is why I added the counter, to make sure that metro is working correctly. Actually the toggle works fine to control the metro, it is just that one can‘t see the cross inside if it is on.

On my PC this patch records the input of the adc~ and shows it continuously in the array. On the iPad the array doesn‘t change at all.
